From 726602e7664015df4d28b8b062cf10d0df70ea1b Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Thu, 7 May 2020 15:29:15 +0800
Subject: [PATCH] arm64: dts: rockchip: remove edp_hpd pinctrl for rk3399 linux

The edp_hpd pin had the conflict with the hdmi cec, enable the hdmi-cec
by default.

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Change-Id: Id129364c352377c59f2b2ff48dd97ffefaf6af11
---
 arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
index 89b915a3d6e2..54479ae63e82 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
@@ -189,6 +189,11 @@
 	};
 };
 
+&edp {
+	/delete-property/pinctrl-names;
+	/delete-property/pinctrl-0;
+};
+
 &iep {
 	status = "okay";
 };
-- 
2.35.3

